content/docs/guides/blackbox-exporter.md

```diff
@@ -0,0 +1,151 @@
+---
+title: Probing HTTP endpoints using the Blackbox Exporter
+---
+
+# Probing HTTP endpoints using the Blackbox Exporter
+
+The Prometheus [**Blackbox Exporter**](https://github.com/prometheus/blackbox_exporter) probes service endpoints for a variety of protocols---HTTP, HTTPS, DNS, TCP, and ICMP---and exposes probe-related metrics to Prometheus.
+
+In this guide, you will:
+
+* Create and start up a simple HTTP server that exposes a health check endpoint
+* Configure Prometheus to scrape metrics from the Blackbox Exporter
+* Configure a Blackbox Exporter to probe that health check endpoint and expose metrics to Prometheus
+
+## Prerequisites
+
+In order to follow along with this guide, you'll need to have the following installed:
+
+* [Python](https://www.python.org) and [pip](https://pypi.org/project/pip)
+* the [Flask](http://flask.pocoo.org) web framework, which can be installed using pip:
+
+  ```shell
+  pip install flask
+  ```
+
+## Local web server with health check
+
+We'll start by creating a simple HTTP server that exposes a single `/health` endpoint. If the server is running, i.e. "healthy," a `GET` request to that endpoint will return a `200 OK` response. The server will be written in Python and use the [Flask](https://pypi.org/project/pip) web framework. Paste this code into a `server.py` file:
+
+```python
+from flask import Flask
+
+app = Flask(__name__)
+
+@app.route('/health')
+def health():
+    return '', 200
+
+if __name__ == '__main__':
+    app.run(port=2112)
+```
+
+To start up the server:
+
+```shell
+python server.py
+```
+
+You should see something like this in the logs:
+
+```
+Starting up the server on port 2112
+ * Serving Flask app "server" (lazy loading)
+ * Environment: production
+   WARNING: Do not use the development server in a production environment.
+   Use a production WSGI server instead.
+ * Debug mode: off
+ * Running on http://127.0.0.1:2112/ (Press CTRL+C to quit)
+```
+
+To test the `/health` endpoint:
+
+```shell
+curl -i http://localhost:2112/health
+```
+
+If you get a `200 OK` response then the HTTP service is ready to go.
+
+# Installing, configuring, and running Prometheus
+
+To install Prometheus, [download the latest release](/download#prometheus) for your platform and untar it:
+
+```bash
+wget https://github.com/prometheus/prometheus/releases/download/v*/prometheus-*.*-amd64.tar.gz
+tar xvfz prometheus-*.*-amd64.tar.gz
+cd prometheus-*.*-amd64
+```
+
+There is a `prometheus.yml` config file in that directory that you can modify to make Prometheus scrape metrics from the Blackbox Exporter. Replace the contents of that file with this:
+
+```yaml
+scrape_configs:
+- job_name: blackbox
+  metrics_path: /probe
+  scrape_interval: 5s
+  params:
+    module: [http_2xx]
+  static_configs:
+  - targets:
+    - localhost:2112/health
```

Comment:
It'd be safest to quote this, it's less likely the user will have issues later as they adjust it